_modules/keystone_policy.py

```diff
@@ -0,0 +1,72 @@
+import apt
+import io
+import json
+import logging
+
+LOG = logging.getLogger(__name__)
+
+
+def __virtual__():
+    return True
+
+
+def rule_list(path, **kwargs):
+    try:
+        file_handle = io.open(path, 'r')
```

Comment:
Using 'with open(path) as file_handler:' seems more pythonistic way and will handle file_handler.close() in finally block. Moreover io.open is an alias for open in python3